docs/dev-proc/source-code-structure.md

```diff
@@ -4,41 +4,33 @@ Every repository forked and maintained by Dasharo Release Team has following
 branch structure:
 
 * `master` - follows upstream project master branch
```

Comment:
```suggestion
* `master` or `main` - follows upstream project master or main branch
```

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020